### ISSUE TITLE
regression: Pressable without functional style or children will not call onPress() on select events

### ISSUE THREAD

**Issue by douglowder** [react-native-tvos/react-native-tvos#744]
### Description

Observed this regression in RNTV 0.74.2-0, in a Pressable with neither a functional form of `style`, or of `children`. The regression is due to https://github.com/facebook/react-native/pull/44615 (https://github.com/react-native-tvos/react-native-tvos/commit/3f71fa8c61087d34106ac5a1ad435eab8ba20e72). Specifically, the changes in lines 387 and 392 should be reverted in order to fix the issue.

### Steps to reproduce

Test case to reproduce the issue: https://github.com/douglowder/PressableExample/blob/2aab5be36424ed02a702e32c7d1f42b975ba7332/App.tsx#L80

### React Native Version

0.74.2-0

### Affected Platforms

Runtime - iOS

### Output of `npx react-native info`

```text
RNTV 0.74.2-0
```

### Stacktrace or Logs

```text
No crasher
```

### Reproducer

https://github.com/douglowder/PressableExample/blob/2aab5be36424ed02a702e32c7d1f42b975ba7332/App.tsx#L80

**Comment by douglowder** [comment]
@Zahoq FYI

**Comment by Zahoq** [comment]
damn, you're right, thank you for pointing that out 🙏 Do we have to verify `focused` in part below?
```ts
      if (focused && evt.eventType === 'select') {
        // $FlowFixMe[incompatible-exact]
        onPress && onPress(evt);
      }
      // $FlowFixMe[prop-missing]
      if (focused && evt.eventType === 'longSelect') {
        // $FlowFixMe[incompatible-exact]
        onLongPress && onLongPress(evt);
      }
```
Pressable is called for that particular `viewTag`, so `pressableTVFocusEventHandler` shouldn't be called for any other `Pressable`

**Comment by douglowder** [comment]
@Zahoq I'll test your proposed fix as well

**Comment by douglowder** [comment]
Thanks @Zahoq I tested your proposed fix and it works fine -- merging 😄

**Comment by Zahoq** [comment]
@douglowder awesome! thanks for info! 🙌

**Comment by douglowder** [comment]
Fixed in 0.74.3-0.
